rcap/cache.go

```diff
@@ -119,3 +126,13 @@ func (m *memoryCache) Delete(key string) error {
 
 	return nil
 }
+
+func defaultCacheRules() CacheRules {
+	c := CacheRules{
+		AllowSet:    true,
+		AllowGet:    true,
+		AllowDelete: true,
+	}
+
+	return c
+}
```

Comment:
I'm not 100% sold on fully permissive by default, but I guess it's nice to keep the Directive.yaml less verbose.

Reply:
Yes, for now we need to do this to make current Directives backwards-compatible, but we may change that behaviour in the future.